### ISSUE TITLE
Gourmet and VMWare have unreadable text in the dark theme.

### ISSUE THREAD

**Issue by izznogooood** [godlyranchdressing/United-GNOME#117]
Thanks for this theme, really good work!

I have some problems with vmware and gourmet like the embedded picture.

![gourmet-united-bug](https://user-images.githubusercontent.com/23040578/27147978-f8375a02-513e-11e7-8c98-28119149a139.png)

![screenshot from 2017-06-14 20-23-56](https://user-images.githubusercontent.com/23040578/27148167-8f9ac24e-513f-11e7-91b1-7685b44ac919.png)

Then theres also the green toggle bars that really should be blue in arch?

![screenshot from 2017-06-14 20-22-01](https://user-images.githubusercontent.com/23040578/27148022-24d94052-513f-11e7-83b8-ff257f3d73c3.png)

Lastly, where can i throw some money your way? I use this theme on all my machines. And its beautiful!

**Comment by godlyranchdressing** [comment]
Thanks!

Both Gourmet and VMWare seem to be doing some kind of overriding that doesn't let them play nicely with dark themes. I did some hacking around to no avail and the messed up text and background colors persist on other dark themes as well. I'll see what I can do, but I don't expect to get lucky.

That green is intentional. The green, red and I think yellow contextual colors are the same in every variant. The only colors that change between the variants are the distro specific primary and secondary colors. Blue and gold in Arch's case. Orange and purple in Ubuntu's.

I was thinking about this for a while and Patreon seems too expectant so I just decided on [PayPal](https://www.paypal.me/APapin). Thanks a lot for even considering it! ~~Guess I should add my real name to my profile already~~
